mobile/test/trade_test.mocks.dart

```diff
@@ -0,0 +1,208 @@
+// Mocks generated by Mockito 5.3.2 from annotations
+// in get_10101/test/trade_test.dart.
+// Do not manually edit this file.
```

Comment:
If this file is generated, should it be in git then?

Reply:
I think it depends on where we want to run these tests - here's a [stackupverflow discussion](https://stackoverflow.com/questions/67927401/should-we-source-control-the-mocks-dart-files-created-by-build-runner-for-mocki) where the recommendation is `No`.

If we want to run these tests e.g. on CI I would check the files in, because another `build` step to generate them would add time to CI runs.
If we only run these tests locally then 

So far these widget tests are not really long-running, but we were not planning to have extensive CI runs with them.

If we only use these tests locally or in dedicated test runs we can ensure to run the build step ` flutter pub run build_runner build` by ourselves or as part of the task.
So, for now I'm happy to add them to the ignore file - we just have to be aware that an additional, manual step might be necessary to run the tests locally.

Reply:
I opted for ignoring it and automatic re-generation with `just flutter-test`